cogs/utils/drawdota.py

```diff
@@ -721,6 +722,44 @@ def get_datetime_cell(match, region_data):
 		TextCell(str_time, font_size=18, horizontal_align="center")
 	)
 
+async def draw_meta_table(sorted_heroes, heroes): 
+	"""Takes a list of [heroes+p/b%+wr%] and draws a nice little discord-friendly table"""
+	
+	border_size = 10
+	grey_color = "#BBBBBB"
+	table = Table(background=discord_color2)
+	#Header
+	headers=[
+		TextCell(""), 
+		TextCell("Hero", padding=0), 
+		TextCell("Win %"), 
+		TextCell("Pick/Ban %")
+	]
+
+	table.add_row(headers)
+	for cell in table.rows[0]: 
+		cell.background = discord_color1
```

Comment:
You should be able to do this on creation of the header cells above by doing `TextCell("", background=discord_color2)`

Reply:
```
	#Header
	headers=[
		TextCell("", background=discord_color2), 
		TextCell("Hero", background=discord_color2, padding=0), 
		TextCell("Win %", background=discord_color2), 
		TextCell("Pick/Ban %", background=discord_color2)
	]
	table.add_row([ColorCell(color=discord_color1, height=6) for i in range(len(headers))])
	first = True
```

This seems to break - maybe I'm misunderstanding. 
![image](https://user-images.githubusercontent.com/30449776/122845311-0dfee480-d2c9-11eb-81f5-2ef3dfee1cbb.png)

Reply:
Hmm, this worked for me?
```
headers=[
TextCell("Hero", background=discord_color1), 
TextCell("", padding=0, background=discord_color1), 
TextCell("Win %", background=discord_color1), 
TextCell("Pick/Ban %", background=discord_color1)
]

table.add_row(headers)
```
maybe you accidentally removed `table.add_row(headers)`?

Reply:
whoops, looks like it. Just fixed it.